tools/syz-symbolize/symbolize.go

```diff
@@ -69,7 +69,7 @@ func main() {
 		}
 		fmt.Printf("TITLE: %v\n", rep.Title)
 		fmt.Printf("CORRUPTED: %v (%v)\n", rep.Corrupted, rep.CorruptedReason)
-		fmt.Printf("MAINTAINERS: %v\n", rep.Maintainers)
+		fmt.Printf("MAINTAINERS: %v\n", rep.Recipients.GetEmails(true))
```

Comment:
We need to print all email here in some form.
This utility is used for manual testing. If one runs, they need to be able to access whom this report will be mailed to (and now who will be in To: and who will be in Cc:).